config/routes.rb

```diff
@@ -11,6 +11,7 @@
   post "/feedback" => "feedback#create"
 
   match "/auth/:provider/callback" => "auth#oauth_callback", via: [:get]
+  get "/auth/failure" => "auth#failure"
```

Comment:
would it make sense to mock oauth in a way that we direct a user to this endpoint on login?